plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/BrowseAssets.js

```diff
@@ -0,0 +1,209 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/*global Ext, NX*/
+
+/**
+ * Browse controller.
+ *
+ * @since 3.0
+ */
+Ext.define('NX.coreui.controller.BrowseAssets', {
+  extend: 'NX.controller.Drilldown',
+  requires: [
+    'NX.Bookmarks',
+    'NX.Conditions',
+    'NX.Permissions',
+    'NX.I18n'
+  ],
+  masters: [
+    'nx-coreui-browseassetfeature nx-coreui-browse-repository-list',
+    'nx-coreui-browse-asset-list'
+  ],
+  stores: [
+    'Asset'
+  ],
+  models: [
+    'Asset'
+  ],
+
+  views: [
+    'browse.BrowseAssetFeature',
+    'browse.BrowseAssetList',
+    'browse.BrowseRepositoryList'
+  ],
+
+  refs: [
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Avoid trailing comma in array and object literals. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ATrailingComma)